data/mods/DinoMod/monsters/dinosaur.json

```
@@ -1251,5 +1251,54 @@
     "anger_triggers": [ "STALK", "PLAYER_WEAK", "HURT" ],
     "placate_triggers": [ "MEAT" ],
     "categories": [ "DINOSAUR" ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_qianzhousaurus",
+    "name": { "str_sp": "Qianzhousaurus" },
+    "species": "DINOSAUR",
+    "default_faction": "qianzhousaurus",
+    "symbol": "d",
+    "color": "white_brown",
+    "volume": "5000 L",
+    "weight": "767 kg",
+    "bodytype": "elephant",
+    "material": [ "flesh" ],
+    "aggression": 300,
+    "morale": 80,
+    "speed": 110,
+    "dodge": 6,
+    "melee_skill": 10,
+    "melee_dice": 10,
+    "melee_dice_sides": 8,
+    "melee_cut": 16,
+    "armor_bash": 12,
+    "armor_cut": 12,
+    "armor_bullet": 12,
+    "hp": 45,
+    "vision_day": 36,
+    "death_function": [ "NORMAL" ],
+    "zombify_into": "mon_zianzhousaurus",
+    "description": "Also known as pinnochio rex, this bipedal dinosaur is slower than most and covered in fur, it also looks very hungry.",
+    "reproduction": { "baby_egg": "egg_qianzhousaurus", "baby_count": 5, "baby_timer": 23 },
+    "baby_flags": [ "SPRING", "SUMMER" ],
+    "biosignature": { "biosig_item": "feces_bird", "biosig_timer": 3 },
+    "special_attacks": [ [ "Poke", 20 ], { "type": "hit", "cooldown": 10 } ],
```

```suggestion
    "special_attacks": [ { "type": "bite", "cooldown": 5 }, [ "LUNGE", 10 ] ],
```
Match Albertosaurus without its special grab. Don't think poke exists, would need to be defined.